collector/physical_disk.go

```diff
@@ -0,0 +1,275 @@
+//go:build windows
+// +build windows
+
+package collector
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func init() {
+	registerCollector("physical_disk", NewPhysicalDiskCollector, "PhysicalDisk")
+}
+
+var (
+	diskWhitelist = kingpin.Flag(
+		"collector.physical_disk.disk-whitelist",
+		"Regexp of disks to whitelist. Disk name must both match whitelist and not match blacklist to be included.",
+	).Default(".+").String()
+	diskBlacklist = kingpin.Flag(
+		"collector.physical_disk.disk-blacklist",
+		"Regexp of disks to blacklist. Disk name must both match whitelist and not match blacklist to be included.",
```

Comment:
Can we change the flag names from `whitelist` and `blacklist` to `include` and `exclude`, respectively?
See #679 for context.

Reply:
Done in 3c3e40e392c64358e355e2138afc07edae69f511, under PR #803.